javascript/javascript1/week2/formal-fullname.js

```diff
@@ -0,0 +1,29 @@
+
+
+function getFullname (firstname, surname, useFormalName, gender) {
+ if (firstname && surname)  {
+    if (useFormalName) {
+        if (gender === 'male') {
+            return (`"Lord ${firstname} ${surname}"`)
```

Comment:
Parenthensis aren't needed to do string interpolation. The following being `true` proves that:
```js
(`"Lord ${firstname} ${surname}"`) === `"Lord ${firstname} ${surname}"`
```

The parenthensis you are used to see belongs to the `console.log` method:
```js
console.log( `Lord ${firstname} ${surname}"` )
```